### PR TITLE
Mes 3643 communication page extract common components add cat b e

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Refractor components of the contact details page so common components are stored in a separate folder. Cat B + E folder structure created.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/communication-page/cat-be/_tests_/communication.cat-be.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+import { TestSlotAttributes } from '@dvsa/mes-test-schema/categories/B';
+import { TranslateService, TranslateModule } from 'ng2-translate';
+import { PopulateTestSlotAttributes }
+  from '../../../../modules/tests/test-slot-attributes/test-slot-attributes.actions';
+import * as welshTranslations from '../../../../assets/i18n/cy.json';
+import { PrivacyNoticeComponent } from '../../components/privacy-notice/privacy-notice';
+import { CommunicationSubmitInfo } from '../../communication.actions';
+import { Language } from '../../../../modules/tests/communication-preferences/communication-preferences.model';
+
+describe('CommunicationCatBPage', () => {
```
Comment (TomBillingtonUK) [inline src/pages/communication-page/cat-be/_tests_/communication.cat-be.page.spec.ts]: needs a rename

File: src/pages/communication-page/cat-be/communication.cat-be.page.ts [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+  candidateAddress$: Observable<Address>;
+
+  conductedLanguage$: Observable<string>;
+}
+@IonicPage()
+@Component({
+  selector: 'communication-cat-be-page',
+  templateUrl: 'communication.cat-be.page.html',
+})
+export class CommunicationCatBEPage extends PracticeableBasePageComponent implements OnInit {
```
Comment (TomBillingtonUK) [inline src/pages/communication-page/cat-be/communication.cat-be.page.ts]: Should be CatBePage

**Review by TomBillingtonUK — APPROVED** [review]
Looks Good :) Nice Job

**Pull request merged**